netlib/http_semantics.py

```diff
@@ -0,0 +1,23 @@
+class Response(object):
```

Comment:
Ok - I reviewed the netlib PR first which is why I missed pathod's Response class here.